apps/registration/index.coffee

```diff
@@ -1,9 +0,0 @@
-express = require 'express'
-routes = require './routes'
-
-app = module.exports = express()
-
-app.set 'views', "#{__dirname}/templates"
-app.set 'view engine', 'jade'
-
-app.get '/register/:token', routes.acceptInvitation, routes.invalidInvitation
```

Comment:
So invitations actually need to be rethought, now that I look at this.

I thought of putting it back, which I could still do: basically it'd just bypass confirmation. But then what happens if u want to sign up from a different email than what you've been invited?

I guess we'll have to put it back and then do a confirmation dance if you change it.